src/projects/pull-request/pull-request-page.html

```diff
@@ -171,7 +171,8 @@
         diff: String,
         selectedPull: {
           type: Object,
-          observer: '_fetchAjax'
+          observer: '_fetchAjax',
+          notify: true
```

Comment:
You mean #223, do you mean you do not want to do the array call in total. Then the multiple assignees are totally broke.

Atm nothing seems to be broken, but that might be due to some if checks I added.